### ISSUE TITLE
Can't create a new collection without a description

### ISSUE THREAD

**Issue by mazameli** [metabase/metabase#7750]
This is in the collections-project branch. If I try to create a collection without a description I get these errors, and nothing user-facing happens in the UI:

JS console logs:
```
warning.js:33 Warning: Failed prop type: The prop `onClose` is marked as required in `ModalContent`, but its value is `undefined`.
    in ModalContent (created by CollectionEditorForm)
    in CollectionEditorForm (created by ReduxForm(CollectionEditorForm))
    in ReduxForm(CollectionEditorForm) (created by Connect(ReduxForm(CollectionEditorForm)))
    in Connect(ReduxForm(CollectionEditorForm)) (created by ReduxFormConnector(CollectionEditorForm))
    in ReduxFormConnector(CollectionEditorForm) (created by ConnectedForm)
    in ConnectedForm (created by CollectionCreate)
    in CollectionCreate (created by CollectionCreate)
    in CollectionCreate (created by Navbar)
    in div
    in OnClickOutsideWrapper
    in div
    in CSSTransitionGroupChild (created by TransitionGroup)
    in span (created by TransitionGroup)
    in TransitionGroup (created by CSSTransitionGroup)
    in CSSTransitionGroup
warning.js:33 Warning: Unknown props `initialValue`, `autofill`, `onUpdate`, `valid`, `invalid`, `dirty`, `pristine`, `error`, `touched`, `visited`, `autofilled` on <input> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in input (created by Input)
    in Input (created by CollectionEditorForm)
    in div (created by FormField)
    in FormField (created by CollectionEditorForm)
    in div (created by CollectionEditorForm)
    in div (created by ModalBody)
    in div (created by ModalBody)
    in ModalBody (created by ModalContent)
    in div (created by ModalContent)
    in ModalContent (created by CollectionEditorForm)
    in CollectionEditorForm (created by ReduxForm(CollectionEditorForm))
    in ReduxForm(CollectionEditorForm) (created by Connect(ReduxForm(CollectionEditorForm)))
    in Connect(ReduxForm(CollectionEditorForm)) (created by ReduxFormConnector(CollectionEditorForm))
    in ReduxFormConnector(CollectionEditorForm) (created by ConnectedForm)
    in ConnectedForm (created by CollectionCreate)
    in CollectionCreate (created by CollectionCreate)
    in CollectionCreate (created by Navbar)
    in div
    in OnClickOutsideWrapper
    in div
    in CSSTransitionGroupChild (created by TransitionGroup)
    in span (created by TransitionGroup)
    in TransitionGroup (created by CSSTransitionGroup)
    in CSSTransitionGroup
api.js:134 POST http://localhost:3000/api/collection 400 (Bad Request)
entities.js:126 metabase/entities/collections/CREATE failed: 
{status: 400, data: {…}, isCancelled: false}
data
:
{errors: {…}}
isCancelled
:
false
status
:
400
__proto__
:
Object

  | _callee$ | @ | entities.js:126
-- | -- | -- | --
  | tryCatch | @ | runtime.js:65
  | invoke | @ | runtime.js:303
  | prototype.(anonymous function) | @ | runtime.js:117
  | step | @ | entities.js:19
  | (anonymous) | @ | entities.js:19
  | Promise.then (async) |   |  
  | step | @ | entities.js:19
  | (anonymous) | @ | entities.js:19
  | (anonymous) | @ | entities.js:19
  | (anonymous) | @ | entities.js:127
  | _callee$ | @ | redux.js:19
  | tryCatch | @ | runtime.js:65
  | invoke | @ | runtime.js:303
  | prototype.(anonymous function) | @ | runtime.js:117
  | step | @ | redux.js:13
  | (anonymous) | @ | redux.js:13
  | (anonymous) | @ | redux.js:13
  | (anonymous) | @ | redux.js:29
  | (anonymous) | @ | store.js:17
  | (anonymous) | @ | bindActionCreators.js:3
  | _callee$ | @ | CollectionCreate.jsx:21
  | tryCatch | @ | runtime.js:65
  | invoke | @ | runtime.js:303
  | prototype.(anonymous function) | @ | runtime.js:117
  | step | @ | CollectionCreate.jsx:9
  | (anonymous) | @ | CollectionCreate.jsx:9
  | (anonymous) | @ | CollectionCreate.jsx:9
  | (anonymous) | @ | CollectionCreate.jsx:20
  | doSubmit | @ | handleSubmit.js:31
  | handleSubmit | @ | handleSubmit.js:64
  | handleSubmit | @ | createHigherOrderComponent.js:195
  | boundFunc | @ | ReactErrorUtils.js:63
  | ../../../node_modules/react-dom/lib/ReactErrorUtils.js.ReactErrorUtils.invokeGuardedCallback | @ | ReactErrorUtils.js:69
  | executeDispatch | @ | EventPluginUtils.js:83
  | executeDispatchesInOrder | @ | EventPluginUtils.js:106
  | executeDispatchesAndRelease | @ | EventPluginHub.js:41
  | executeDispatchesAndReleaseTopLevel | @ | EventPluginHub.js:52
  | forEachAccumulated | @ | forEachAccumulated.js:22
  | processEventQueue | @ | EventPluginHub.js:252
  | runEventQueueInBatch | @ | ReactEventEmitterMixin.js:15
  | handleTopLevel | @ | ReactEventEmitterMixin.js:25
  | handleTopLevelImpl | @ | ReactEventListener.js:70
  | perform | @ | Transaction.js:141
  | batchedUpdates | @ | ReactDefaultBatchingStrategy.js:60
  | batchedUpdates | @ | ReactUpdates.js:95
  | dispatchEvent
```

Server logs:

```
05-29 10:00:56 DEBUG metabase.middleware :: GET /api/user/current 200 (2 ms) (1 DB calls). Jetty threads: 8/50 (7busy, 2 idle, 0 queued)
05-29 10:00:56 DEBUG metabase.middleware :: GET /api/session/properties 200 (2 ms) (1 DB calls). Jetty threads: 8/50 (6 busy, 3 idle, 0 queued)
05-29 10:00:56 DEBUG metabase.middleware :: GET /api/collection 200 (2 ms) (1 DB calls). Jetty threads: 8/50 (6 busy, 3 idle, 0 queued)
05-29 10:00:56 DEBUG metabase.middleware :: GET /api/collection/root 200 (4 ms) (3 DB calls). Jetty threads: 8/50(6 busy, 3 idle, 0 queued)
05-29 10:00:56 DEBUG metabase.middleware :: GET /api/setting 200 (275 µs) (0 DB calls). Jetty threads: 8/50 (6 busy, 3 idle, 0 queued)
05-29 10:01:18 DEBUG metabase.middleware :: POST /api/collection 400 (968 µs) (0 DB calls).
{:errors {:description "value may be nil, or if non-nil, value must be a non-blank string."}}

05-29 10:01:20 DEBUG metabase.middleware :: POST /api/collection 400 (712 µs) (0 DB calls).
{:errors {:description "value may be nil, or if non-nil, value must be a non-blank string."}}
```

**Comment by camsaul** [comment]
this has been fixed for almost a month in the `nested-collections` branch we should really just merge that :D

**Comment by mazameli** [comment]
Fixed by #7753
